modules/administration-guide/pages/installing-image-puller-on-openshift-using-the-web-console.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {image-puller-name-short} on OpenShift using the web console
+:keywords: administration-guide, image-puller, openshift, operator, operatorhub
+:page-aliases: .:installing-image-puller-on-openshift-using-operatorhub, installing-image-puller-on-openshift-using-operatorhub
+
+[id="installing-image-puller-on-openshift-using-the-web-console_{context}"]
+= Installing {image-puller-name-short} on OpenShift using the web console
+
+This procedure describes how to install the {image-puller-operator-name} on OpenShift using the OpenShift web console.
```

Comment:
Replace multiple instances of `This procedure describes how to` with `You can`...